### PR TITLE
Overhauled default zombie drops

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: Balance "Fixed zombies dropping inappropriate stuff on death."```  

#### Purpose of change
Currently list of items dropped on zombies' death has several problems. Specifically, all of them come from the fact that `default_zombie_drops` uses `livingroom`, `kitchen`, `bedroom` and some other items groups, which is inappropriate for zombie drops.
First, it allows zombies to drop big items which they couldn't store when they were alive (like television sets or lawnmower), and even if they wielded these things in hands, when they became zombies, they would certainly drop them to the ground.
Second, it allows zombies to drop items small enough to be carried in pockets or in bags (like hockey puck, lye powder, pot, or pasta extruder), but in my opinion it is unlikely that anyone would ever carry such things with them to the street.

#### Describe the solution
To address these problems I:
- removed aforementioned `livingroom`, `kitchen`, `bedroom` item groups from `default_zombie_drops` monster drops;
- added two new item groups (the one for the common people everyday handheld and pocket-carried items, and the other one for the common people everyday bag-carried items) and added them to `default_zombie_items`.

`default_zombie_items_pockets` contains items that one could have carried in his pockets prior to zombification: pocket knives, smoking sticks (all variants like cigars, electronic cigarettes, joints etc), lighters or matches, small electronic devices (mp3-players, phones etc), snacks and so forth;
`default_zombie_items_bags` contains items that only fit in bags and thus will spawn only with bags. The examples of items are laptops, thermos, telescopic umbrella, small house tools (screwdrivers, pliers, wrench etc), bottles and cans with drinks and so forth.

I tried to carefully move most of the items from old item groups into these two new item groups. But many items weren't moved (such as microwaves, kilns, pucks, basketballs, pots, pressure cookers and the like) which I find too big or inappropriate for zombies to carry. In general variety of items that zombies can drop on their death is significantly reduced. To compensate that I increased chance for random items from two new item groups (aside from clothing and a cash card) to spawn from 50% to 75%. Hence the `[CR]` tag. I'm also open to suggestions on items that could be added to or removed from the new item groups.

#### Describe alternatives you've considered
Leave the current system as is and continue to watch zombies drop mini-fridges and dehydrators on death.

### CODE REVIEW

**Comment by curstwist** [comment]
Bags as part of zombie clothing is fairly rare. Wonder how this will impact getting tools. Maybe the recent changes to npc clothing options will even it out.

**Comment by Night-Pryanik** [comment]
>Bags as part of zombie clothing is fairly rare.

This PR makes it so zombies drop bags roughly with the same frequency as they don't drop bags.

File: data/json/item_groups.json [hunk 1]
```diff
@@ -340,7 +340,6 @@
       ["gum", 30],
       ["caff_gum", 3],
       ["flask_hip", 5],
-      ["sm_extinguisher", 10],
```
Comment (jbytheway) [inline data/json/item_groups.json]: Was it intentional to remove `sm_extinguisher` from `living_room`?

Comment (Night-Pryanik) [reply data/json/item_groups.json]: Yes, it is double of `sm_extinguisher` on line 315.

**Comment by Night-Pryanik** [comment]
jenkins rebuild

**Comment by kevingranade** [comment]
Squashed and merged into 0.D and master.

**Pull request closed without merging**